tests/__snapshots__/test_utils/test_bom_generation_altium_with_device_sheets.csv

```diff
@@ -0,0 +1,39 @@
+Comment,Designator,Name,Quantity
+61300211121,P4,Header_Male_1R_02P,1
+61300211121,P3,Header_Male_1R_02P,1
+61300211121,P7,Header_Male_1R_02P,1
+61300211121,P6,Header_Male_1R_02P,1
+61300211121,P1,Header_Male_1R_02P,1
+61300211121,P8,Header_Male_1R_02P,1
+61300211121,P2,Header_Male_1R_02P,1
+61300211121,P5,Header_Male_1R_02P,1
+LM2678SX-3.3,U1,CMP-0069-00188-2,1
+SBR1045D1-13,D1,3c54d775813915f30481cfffa5e9369,1
```

Comment:
one thing we might need to follow up on is that there are multiple "D1"s (for each of the sub-references). 
Altium splits these definitions into ["PhysicalDesignator" and "LogicalDesignator"](https://hub.allspice.io/AllSpice-Demos/Altium-Device-Sheet-Usage-Demo/src/commit/db729cf7f701d7e1f4cda691be67575b2e56ea0e/DCDC%20Regulators%20Breakout/DCDC%20Regulators%20Breakout.Annotation#L12) (though it's not clear how this schema works to me). 
This is probably fine for now, but we should add this to the list of things to come back to. BO might have issues.
Altium has an "Annotate Compiled Sheets" tool. I couldn't get it to work. @polypour is this what you were working with? Did you generate a BOM we can compare to?
<img width="882" alt="image" src="https://github.com/user-attachments/assets/675cf509-3d6c-4ccd-af62-dc0d71ac9430">

Reply:
I think this is part of what @polypour was referring to when he mentioned that he wasn't able to re-number all the references. I can re-generate the BOM if/when we figure that out and see what we get!

Reply:
Yeah, agreed. Unfortunately, I suspect that it's not going to be as simple as updating the reference designators in the Altium test design and having that produce the same BOM output as Altium.

Since the device sheets are shared between multiple "parent" projects, the designators in the sub-sheets can't be shared in all of those parent instances (there will always be the potential for collisions). EITHER, the designators are mapped from the sub-sheet to the parent sheet (which is what I _think_ the `.Annotation` file does) OR there is some suffix appended.

As discussed, we need to continue with what we have to get some feedback on the initial implementation, but I'll create a ticket to come back to this.